requirements.legacy.txt

```diff
@@ -0,0 +1,61 @@
+# These are the base Weave requirements, enough for weave tracking and evaluation
```

Comment:
This can be severely trimmed down - much of this is trace specific. fine to be a followup

Reply:
Is the minimal set described here?  If yes we can replace:
https://github.com/wandb/core/blob/master/services/weave-python/requirements.txt